components/automate-chef-io/data/docs/api-static/04-meta-auth.json

```diff
@@ -0,0 +1,10 @@
+{
+  "securityDefinitions": {
+    "APIToken": {
```

Comment:
we have only included the security definition for API tokens. there is no easy way to get a Bearer token so we'd rather encourage users to stick to API tokens.

Reply:
Ok. I limited the language so we say "Authenticate with an API token". If we're not going to discuss other methods, then let's not refer to them.